scripts/zones/Sealions_Den/mobs/Mammet-22_Zeta.lua

```diff
@@ -20,21 +19,39 @@ local forms =
     STAFF   = 3,
 }
 
+local tpMoves =
```

Comment:
For this and the table above, would suggest breaking this out to single items per line, and commenting there